2023-03-12 13:29:47,610 [client.py:234] [INFO]: Attempting to connect to server: localhost 
2023-03-12 13:29:48,953 [MainMenu.py:109] [INFO]: Connected to localhost 
